src/org/traccar/protocol/Gt06ProtocolDecoder.java

```diff
@@ -202,7 +205,11 @@ private boolean decodeLbs(Position position, ChannelBuffer buf, boolean hasLengt
                 buf.readUnsignedShort(), buf.readUnsignedByte(), buf.readUnsignedShort(), buf.readUnsignedMedium())));
 
         if (length > 0) {
-            buf.skipBytes(length - 8);
+            if (hasLength && !isExtended) {
```

Comment:
Why do we need `isExtended`? Should this just be:
```java
buf.skipBytes(length - (hasLength ? 9 : 8));
```

Reply:
I added this because of the test case is failing for Extended decode for 7979.
If I wont add this then one of the test case is failing. As this is true for 7878 as per the documentation I have for GT06.

Reply:
Extended option has extra RSSI field after standard LBS data. I propose we add it directly into `decodeExtended` code.

Reply:
Sure thats like a good suggestion. I will change this.

Reply:
I have made the changes. Please check.